### DISCUSSION TITLE
Has OAuth 1.0 support been completely dropped in 5-beta?

### DISCUSSION THREAD

**Discussion by loan-laux** [nextauthjs/next-auth#10438] [Help]
As the title says. Is there any way to still use OAuth 1.0 with 5-beta or should I stick to v4?

**Comment by julbrs** [comment]
There is still a lot of provider that are using OAuth 1.0a (SmugMug for example: https://api.smugmug.com/api/v2/doc/tutorial/authorization.html). It would be nice to still have active support on that!
